### DISCUSSION TITLE
5주차 반성문

### DISCUSSION THREAD

**Discussion by lcqff** [BDD-CLUB/challenge-every-week-posting#83] [6. 반성문]
2주 연속으로 글을 쓰지 않았습니다
하지만 절 믿어주십시오
글 쓸 것이 많습니다 
종강하고 일주일동안 해이해져서 미루고 있던 건 맞습니다. 헬스 등록해놓고 운동도 안갔습니다(ㅋㅋ)
하지만 절 믿어주십시오
다음주면 카테캠때문에 강제적으로 갓생을 살아야합니다
다음주에 글 3개... 공약합니다!
그리고 운동도 가겠습니다
원래 학교때문에 40분밖에 운동 못했는데
이제 학교도 안 가고 이동시간도 5분이니까 런닝머신뿐만 아니라 헬스장 기구를 조저볼 계획입니다!!
근력운동입니다! 체력은 근육에서 나오는 겁니다!!
웜업40분! 근력 30분? 근력운동 안해봐서 시간이 얼마나 걸리는진 모릅니다!
한혜진의 헬린이 루틴 보고 따라할겁니다! 
아무튼 다음주! 다음주의 저를 믿어주십시오! 
글 쓰곤 있는데 오늘 올라갈 글은 아닙니다! 진짭니다!

**Comment by pnudpwls1103** [comment]
아즈아!!

**Comment by publdaze** [comment]
화이팅화이팅
